src/latticpp/ckks/keygen.cpp

```diff
@@ -11,10 +11,55 @@ namespace latticpp {
         return KeyGenerator(lattigo_newKeyGenerator(params.getRawHandle()));
     }
 
+    SwitchingKey getRotationKey(const RotationKeys &rotationKeys, uint64_t galEl) {
+      return SwitchingKey(lattigo_getRotationKey(rotationKeys.getRawHandle(), galEl));
+    }
+
+    void setRotationKey(const RotationKeys &rotKeys, const SwitchingKey &rotKey, uint64_t galEl) {
```

Comment:
`rotKey`->`switchKey`, and name this `setSwitchingKey`?

Reply:
yes